scripts/openmc-get-photon-data

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+"""
+Download ENDF/B-VII.1 ENDF data from NNDC for photo-atomic and atomic
+relaxation data and convert it to an HDF5 library for use with OpenMC.
+This data is used for photon transport in OpenMC.
+"""
+
+import os
+import sys
+import shutil
+import zipfile
+import argparse
+from io import BytesIO
+from urllib.request import urlopen
+
+import openmc.data
+
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+    formatter_class=CustomFormatter
+)
+parser.add_argument('-c', '--cross-sections',
+                    help='cross_sections.xml file to append libraries to')
+args = parser.parse_args()
+
+base_url = 'http://www.nndc.bnl.gov/endf/b7.1/zips/'
+files = ['ENDF-B-VII.1-photoat.zip', 'ENDF-B-VII.1-atomic_relax.zip']
+block_size = 16384
+
+# ==============================================================================
+# DOWNLOAD FILES FROM NNDC SITE
+
+if not os.path.exists('photon_hdf5'):
+    os.mkdir('photon_hdf5')
+
+for f in files:
+    # Establish connection to URL
+    url = base_url + f
```

Comment:
I believe it is best practice to wrap `urlopen` in a `with` clause, since it can potentially error out and leave a socket open for the request.  E.g.,:

```
with urlopen(url) as response:
   file_size = <...>
```

This howto page has some details:
https://docs.python.org/3/howto/urllib2.html

Additionally, I think `urllib.request.urlopen` actually returns a http.client.HTTPResponse object [1]?  `res` or `response` might be a more readable variable name.

[1] https://docs.python.org/3/library/urllib.request.html#urllib.request.urlopen